lhotse/cut.py

```diff
@@ -0,0 +1,68 @@
+from dataclasses import dataclass
+from typing import Dict, List
+
+from lhotse.features import Features
+from lhotse.supervision import SupervisionSegment
+from lhotse.utils import Seconds
+
+
+# (PZ): After writing this, I'm thinking of making Cut an interface that might not necessarily expose
+#  begin, end and channel (duration could stay I guess, ID I'm not sure about yet).
+#  Then, there would be 2 (or more?) classes satisfying the interface: SingleCut and MultiCut.
+#  That would make implementation of all the manipulations easier, I guess.
+
+# I'm striving for maximally "lazy" implementation, e.g. when overlaying Cuts, I'd rather sum the feature matrices
+#  only when somebody actually calls "load_features". Actually I might remove "features" from the interface too
+#  for that reason.
+
+
+@dataclass
+class Cut:
+    id: str
+
+    # (PZ): I'm not super sure if this should be just one channel... but it's certainly easier to start this way
+    channel: int
+
+    # Begin and duration are needed to specify which chunk of features to load.
+    begin: Seconds
+    duration: Seconds
+
+    supervisions: List[SupervisionSegment]
+
+    # The features can span longer than the actual cut.
```

Comment:
Maybe we can note here that the Features object "knows" its start and end time within
the underlying recording, and that we expect the interval [begin,begin+duration]
to be a superset of the interval represented in `features`.